setup.py

```diff
@@ -14,19 +14,13 @@ def read(fname):
 
 
 install_requires = [
-    'mezzanine >= 3.1.10',
     'django-treebeard',
-    'django-filer>=0.9.6',
-    'South',
+    # pyScss fixed until https://github.com/fusionbox/django-pyscss/pull/26 is fixed.
+    'pyScss==1.2.1',
```

Comment:
Yes, this still needs to be removed.  However, https://github.com/stefanfoulis/django-filer/pull/480 is still unresolved (no 0.9.9) release.